ISISComputingGroup/ibex_gui
# Added OPI and component for the Chell CCD100

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/1827

The OPI has tried to follow the original LabView as much as possible, as such it talks to 4 CCD100 IOCs and graphs their readings on the same graph.

### To test

* Start at least one CCD100 IOC
* Create a CCD100 component in device screen/synoptic
* Confirm OPI connects to IOC and displays information as expected.

---

#### Code Review


### Functional Tests


### Final Steps
